trade-finance-manager-api/src/v1/controllers/update-issued-facilities.js

```diff
@@ -28,7 +28,10 @@ const updatedIssuedFacilities = async (deal) => {
       hasBeenAcknowledged,
     } = facility;
 
-    const shouldUpdateFacility = (hasBeenIssued && !hasBeenAcknowledged);
+    // `hasBeenAcknowledged` is BSS/EWCS specific only
+    const shouldUpdateFacility = dealType === CONSTANTS.DEALS.DEAL_TYPE.GEF
```

Comment:
I'm not sure if you need this here, but we have now `hasBeenIssued` both for GEF and BSS

Reply:
Brilliant, have updated.